### PR TITLE
Ronnie/jenkinsfile cleanup

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,60 +1,64 @@
...
-    }
-        steps {
-         withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
-          sh '''docker login quay.io -u $USER -p $PASS'''
-        }
-          sh '''docker tag  quay.io/ocpmetal/assisted-service:test quay.io/ocpmetal/assisted-service:$(git rev-parse --verify HEAD)'''
-          sh '''docker tag  quay.io/ocpmetal/assisted-service:test quay.io/ocpmetal/assisted-service:latest'''
-          sh '''docker push quay.io/ocpmetal/assisted-service:latest'''
-          sh '''docker push quay.io/ocpmetal/assisted-service:$(git rev-parse --verify HEAD)'''
+		stage('Deploy to prod') {
```
Comment (eranco74) [inline Jenkinsfile:25]: Deploy to Prod??
`Publish image`

Comment (ronniel1) [reply Jenkinsfile]: fixed

**Pull request closed without merging**